### ISSUE TITLE
Empty volume rendering caused by spacing choice

### ISSUE THREAD

**Issue by MPenaR** [pyvista/pyvista#444]
## Description
I trying to do volume rendering with some data that I have in RectilinearGrid format. To do that I'm trying first to create a UniformGrid object. 
```
s = pv.read("reciprocal_total.vtk")
U = pv.UniformGrid()
U.origin = [-0.1, -0.1, -0.1]
U.spacing = [0.2/29, 0.2/29, 0.2/29]
U.dimensions = [ 30, 30, 30 ]
U['DT'] = s['DT_tot']
U.plot(volume=True,notebook=False,opacity='linear_r')
```
However when I ask it to plot it I get an empty image:
![empty](https://user-images.githubusercontent.com/9216537/68316514-e2aa9000-00b9-11ea-9e94-11e4c2471234.png)
If I plot it as a surface

```
p = pv.Plotter(notebook=False, shape=(1, 2), border=False)
p.subplot(0, 0)
p.add_text("Rectilinear", font_size=24)
p.add_mesh(s)
p.subplot(0, 1)
p.add_text("Uniform", font_size=24)
p.add_mesh(U)
p.link_views()
p.show()
```
 it works, and I get the same result as if I plotted the rectilinear grid: 
![surface_works](https://user-images.githubusercontent.com/9216537/68318432-10dd9f00-00bd-11ea-9896-805e4b9b0d69.png)

If I extract the array into a n-dimensional numpy array 
```
U2 = np.reshape(U.get_array('DT'),[30,30,30],'F')
p = pv.Plotter(notebook=False)
p.add_volume(U2,resolution=[100,100,100], opacity='sigmoid_r')
p.show()
```
the volume rendering works (not very pretty but it works) so I guess I am doing something wrong:
![it_works](https://user-images.githubusercontent.com/9216537/68318895-bee94900-00bd-11ea-967e-52bf303abe80.png)

What is what I'm doing wrong in the first code?

## Example Data
The sample data can be found at [reciprocal_total.vtk](https://drive.google.com/open?id=1fNcfrdVrjuc-VNlo2vVWfEslz4JYzOpm)

**Comment by banesullivan** [comment]
@MPenaR, this should be fixed in #464 - If you have time to check out that branch and make sure all is working as you expect, it would be greatly appreciated! Here is what I get now for your data:

```py
import pyvista as pv

s = pv.read("reciprocal_total.vtk")

U = pv.UniformGrid()
U.origin = [-0.1, -0.1, -0.1]
U.spacing = [0.2/29, 0.2/29, 0.2/29]
U.dimensions = [ 30, 30, 30 ]
U['DT'] = s['DT_tot']

p = pv.Plotter(notebook=0)
p.add_volume(U, cmap="viridis", opacity="linear_r",
             mapper="smart", opacity_unit_distance=U.length / 50)
p.show_grid()
p.show()
```

![2019-11-21 23 58 45](https://user-images.githubusercontent.com/22067021/69404595-e4c83d80-0cba-11ea-93a9-83f8accc81ec.gif)

**Comment by MPenaR** [comment]
Works as expected, you are awesome!
